src/TuxAnimations.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-var React = require('react/addons');
+var React = require('tux/React');
 var Arrival = require('arrival');
 var Easings = require('./TuxAnimationEasings');
+var ReactTransitionGroup = require('tux/React/TransitionGroup');
 
-//makeAnimation FUNCTION: creates animation for wrapped component based on props of wrapper component
+//createAnimationClass FUNCTION: creates animation for wrapped component based on props of wrapper component
```

Comment:
Change comment to: `creates animationClass which will animate based on the passed in transitions object`.